### PR TITLE
chore: tag 0.12.5

### PR DESCRIPTION
**Pull request by pjenvey**

### CODE REVIEW

File: syncstorage-common/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[package]
```
Comment (ethowitz) [inline syncstorage-common/Cargo.toml:1]: Do we want this file here? The setting crates PR removed the syncstorage-common crate

Comment (pjenvey) [reply syncstorage-common/Cargo.toml]: Thanks, good catch! I thought this was deleted, merge fail.

**Comment by ethowitz** [comment]
When this PR is merged, will be it be squashed into one commit? If so, that squashed commit would include the MySQL client library update -- is that intended?

**Comment by pjenvey** [comment]
> When this PR is merged, will be it be squashed into one commit? If so, that squashed commit would include the MySQL client library update -- is that intended?

I temporarily enable merge commits for a branch like this (I really don't want them off 100% I just don't want it to by the default for merging PRs)

**Review by ethowitz — APPROVED** [review]

**Pull request merged**